backend/src/main/java/com/ambacam/model/RequeteGroupe.java

```diff
@@ -0,0 +1,118 @@
+package com.ambacam.model;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+@Entity
+public class RequeteGroupe extends AuditingCommonEntity implements Serializable {
+	private static final long serialVersionUID = -909404128014016669L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	private Long id;
+
+	@NotEmpty(message = "The name must not be empty")
+	@NotNull(message = "The name must not be null")
+	@Column(columnDefinition = "varchar", nullable = false)

```

Comment:
If nom is mandatory, please update the swagger

Reply:
In fact, the name and other parameters should be mandatory, but not the description

Reply:
@Coccoonx I think, that the Swagger is up to date. Only the properties of RequeteGroupe should be mandatory.
Did you see anything which is not normal? Please, tell it promptly so that I can change as soon as possible.

The model and repository for Requete appears here, because this issue depends partly on that the Requete's issue. That is why, I implemented it here without going to deep in the service and other stuff. Just the model and the repository are created. Does it solve your problem? If not, I invite you to specify it clearly.